CouplingDataUser.H

```diff
@@ -49,7 +58,10 @@ public:
     void setPatchIDs(std::vector<int> patchIDs);
 
     //- Set the locations type of the interface
-    void setLocationsType(std::string locationsType);
+    void setLocationsType(LocationType locationsType);
+
+    // Check, if the data set supports the interface node location
```

Comment:
No comma before "if" in English:
```suggestion
    // Check if the dataset supports this interface nodes location
```